<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EightDigitPuzzle</class>
 <widget class="QMainWindow" name="EightDigitPuzzle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EightDigitPuzzle</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>img/icon.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>650</height>
    </size>
   </property>
   <widget class="QTextBrowser" name="Text_Information">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>451</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="LCD_Grid">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="LCD_9">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="LCD_3">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="LCD_6">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="LCD_2">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="LCD_5">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="LCD_4">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="LCD_8">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="LCD_7">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="LCD_1">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="Check_GetTree">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Use in-built puzzle
to get a tree view</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Progress_Steps">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>451</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Function">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Manhattan Distance</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Manhattan Distance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euclidean Distance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cosine Distance</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label_SelectFunction">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a heuristic function:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Copyright">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright Â© 2022 LPH. All Rights Reserved</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Lucida Handwriting&quot;;</string>
    </property>
    <property name="text">
     <string>8 Digit Puzzle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>451</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Button_PerviousStep">
       <property name="text">
        <string>&lt;- Pervious step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Auto">
       <property name="text">
        <string>Auto playing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_NextStep">
       <property name="text">
        <string>Next step -&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Button_Init">
       <property name="text">
        <string>Random</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Reset">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Button_Show">
       <property name="text">
        <string>Show tree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ShowStats">
       <property name="text">
        <string>Show stats</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>Text_Information</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>Check_GetTree</zorder>
   <zorder>Progress_Steps</zorder>
   <zorder>Combo_Function</zorder>
   <zorder>Label_SelectFunction</zorder>
   <zorder>Label_Title</zorder>
   <zorder>Label_Copyright</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
